mono/mini/mini.c

```diff
@@ -2872,6 +2872,19 @@ insert_safepoints (MonoCompile *cfg)
 		return;
 	}
 
+	if (cfg->method->wrapper_type == MONO_WRAPPER_MANAGED_TO_NATIVE) {
```

Comment:
Just merge that case with the one above at https://github.com/mono/mono/pull/11789/files/4200cc62bff4c880713c335a2de8f9135e7092ae..b06eb38ee0a4bfefdfdab0911f438a396c453d5f#diff-dbbf26d27078d673bee0d8d752e3b903R2858. 
```
if (info && info->subtype == WRAPPER_SUBTYPE_ICALL_WRAPPER &&
	(info->d.icall.func == poll_func ||
	 info->d.icall.func == mono_thread_interruption_checkpoint ||
 	 info->d.icall.func == mono_threads_exit_gc_safe_region_unbalanced)) { ... }
```

Reply:
And that's funny, that's how the code was before https://github.com/mono/mono/pull/11789/commits/4200cc62bff4c880713c335a2de8f9135e7092ae.